bittensor/__init__.py

```diff
@@ -287,8 +290,14 @@ def debug(on: bool = True):
     ProposalVoteData,
 )
 
+# Allows avoiding name spacing conflicts and continue access to the `subtensor` module with `subtensor_module` name
 from . import subtensor as subtensor_module
-from .subtensor import subtensor as subtensor
+
+# Double import allows using class `Subtensor` by referencing `bittensor.Subtensor` and `bittensor.subtensor`.
+# This will be available for a while until we remove reference `bittensor.subtensor`
+from .subtensor import Subtensor
+from .subtensor import Subtensor as subtensor
```

Comment:
We shouldn't remove the bt.subtensor factory design.

Reply:
This is still under decision. Both options are available now, as we discussed in the chat.